### ISSUE TITLE
[BUG] ACF.GetVersion lua error on clients that don't have the addon installed

### ISSUE THREAD

**Issue by thegrb93** [Stooberton/ACF-3#42]
```
[ERROR] addons/acf-3/lua/acf/client/sk_menu.lua:251: attempt to index local 'Repo' (a nil value)
  1. ACFHomeGUICreate - addons/acf-3/lua/acf/client/sk_menu.lua:251
   2. CreateAttribs - addons/acf-3/lua/acf/client/sk_menu.lua:50
    3. UpdateDisplay - addons/acf-3/lua/acf/client/sk_menu.lua:224
     4. OnNodeSelected - addons/acf-3/lua/acf/client/sk_menu.lua:54
      5. SetSelectedItem - lua/vgui/dtree.lua:95
       6. Init - addons/acf-3/lua/acf/client/sk_menu.lua:59
        7. CreateFromTable - lua/includes/extensions/client/panel/scriptedpanels.lua:62
         8. func - addons/acf-3/lua/weapons/gmod_tool/stools/acfmenu.lua:49
          9. FillViaFunction - gamemodes/sandbox/gamemode/spawnmenu/controlpanel.lua:100
           10. FillViaTable - gamemodes/sandbox/gamemode/spawnmenu/controlpanel.lua:89
            11. ActivateTool - lua/includes/modules/spawnmenu.lua:287
             12. DoClick - gamemodes/sandbox/gamemode/spawnmenu/toolpanel.lua:118
              13. unknown - lua/vgui/dlabel.lua:234
```

**Comment by TwistedTail** [comment]
Is your server using the latest version? There was a small bug that got fixed on 6b41a8e2e6d756df84f9cb5127f1787249a6fc5d and it's related to that line.

**Comment by thegrb93** [comment]
Yeah the code matches that change.

**Comment by TwistedTail** [comment]
It was probably the network message's fault. Tried joining my server without both ACF and ACF Missiles installed and it worked fine. Everything should be fixed on 4e9fed5c0f767d08fe05b98022877cc5a166390f.
